packages/account/src/providers/provider.ts

```diff
@@ -69,6 +69,7 @@ export const RESOURCES_PAGE_SIZE_LIMIT = 512;
 export const TRANSACTIONS_PAGE_SIZE_LIMIT = 60;
 export const BLOCKS_PAGE_SIZE_LIMIT = 5;
 export const DEFAULT_RESOURCE_CACHE_TTL = 20_000; // 20 seconds
+export const GAS_USED_MODIFIER = 1.3;
```

Comment:
Shouldn't it be 20%?

Reply:
@arboleya Yes, @luizstacio suggested using 30%, but after running some tests, I believe it’s better to stick with the initial value of 20%. The issue with using a higher value is that it might cause us to exceed the maximum allowed gas.

Reply:
That can happen with any multiplier, right?

Perhaps a `Math.max(gas, MAX_GAS)` could have a place here.

Reply:
@arboleya I was on that already. I decided to spend some time running some validations https://github.com/FuelLabs/fuels-ts/pull/3319/commits/66b43b129481dc5b092133f6b259451f50137e0e

Reply:
I'm sure you were. 😄